README.md

```diff
@@ -454,6 +454,19 @@ convo, err := intercom.Conversations.Close("1234", &closerAdmin)
 convo, err := intercom.Conversations.Assign("1234", &assignerAdmin, &assigneeAdmin)
 ```
 
+### Webhooks
+
+### Notifications
+
+If you have received a JSON webhook notification, you may want to convert it into real Intercom object. A Notification can be created from any `io.Reader`, typically a http request:
+
+```go
+var r io.Reader
+notif, err := intercom.NewNotification(r)
+```
+
+The returned Notification will contain exactly 1 of the `Compmay`, `Conversation`, `Event`, `Tag` or `User` fields populated.
```

Comment:
`Compmay` -> `Company`

Reply:
Hi @josler can you expand on your comment "the objects embedded in the notification might not be exact equivalents"? I don't quite understand what you mean.

Reply:
Hi @heldtogether 
I think what he was referring to was that in some cases it is not the full object returned in the notification. For example, for the conversation it is only a conversation part I think and not a whole conversation object. So maybe just add a point in the README to clarify this. And then I think we are good to merge this
Thanks again

Reply:
@choran I've updated as advised 😃

Reply:
Great, I will merge it so!
Thanks again for this